test/models/policy_test.rb

```diff
@@ -80,7 +80,7 @@ class PolicyTest < ActiveSupport::TestCase
 
     should 'destroy business objectives without policies on destroy' do
       assert_difference('BusinessObjective.count' => -1) do
-        policies(:one).destroy
+        Policy.where(id: policies(:one)).destroy_all
```

Comment:
@vkrizan crazy, but `destroy_all` has different behavior than `destroy` when it comes to the `changes` and `previous_changes` on the model. :see_no_evil: This change makes the test fail against master.